tla/consensus/ccfraft.tla

```diff
@@ -217,7 +227,7 @@ CurrentTermTypeInv ==
 VARIABLE state
 
 StateTypeInv ==
-    \A i \in Servers : state[i] \in States
+    \A i \in Servers : state[i] \in LeadershipStates
```

Comment:
It probably makes sense to rename `state` to `leadershipState`, or `leadership` (as opposed to `membership`, `membershipState`, or `mState` or whatever), but that may be the next PR.

Reply:
done in https://github.com/microsoft/CCF/pull/5902/commits/db272604b4644eb45a82edd72d1dfd9319df8746